pkg/blockbuilder/blockbuilder.go

```diff
@@ -643,6 +645,22 @@ consumerLoop:
 	return newState, nil
 }
 
+func getBlockCategoryCount(sectionEndTime time.Time, blockMetas []tsdb.BlockMeta) (prev, curr, next int) {
+	// Doing -30m will take care
```

Comment:
(nit) What does this comment say?

Reply:
Whoops looks like I got distracted midway. Will fix it.